SmartDeviceLink/private/SDLStreamingVideoLifecycleManager.m

```diff
@@ -454,6 +453,15 @@ - (void)didEnterStateVideoStreamReady {
 
     [[NSNotificationCenter defaultCenter] postNotificationName:SDLVideoStreamDidStartNotification object:nil];
 
+    if (self.shouldUpdateDelegateOnSizeChange && self.delegate != nil) {
```

Comment:
Just curious why there is a `shouldUpdateDelegateOnSizeChange` flag? I believe `didEnterStateVideoStreamReady` should only be called once the module has ACK the request the start the video stream so I don't think there is any danger of `didEnterStateVideoStreamReady` getting called more than once. Is there some edge case I am not thinking of in relation to this?

Reply:
The `SDLVideoStreamingCapability` may change in 2 cases (2 states): `SDLVideoStreamManagerStateStarting` and `SDLVideoStreamManagerStateReady`. The notification will be sent only in the 1st case and skipped in the 2nd. 
If it is wrong I can remove this check.

Reply:
I see what is going on now. If the video is in the suspended state but has then been resumed we don't want to send a duplicate `videoStreamingSizeDidUpdate` notification to the observer.